cluster/templates/fluo-cluster/conf/yarn-site.xml

```diff
@@ -79,10 +79,10 @@
   </property>
   <property>
     <name>yarn.nodemanager.container-executor.group</name>
-    <value>ec2-user</value>
+    <value>$cluster.username</value>
```

Comment:
It's lower case because it's using the the property name from fluo-deploy.props.